Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -554,9 +576,7 @@ final class CacheAdvanceTests: XCTestCase {
         throws
         -> CacheHeaderHandle
     {
-        if zeroOutExistingFile {
-            FileManager.default.createFile(atPath: testFileLocation.path, contents: nil, attributes: nil)
-        }
+        if zeroOutExistingFile { clearCacheFile() }
```

Comment:
This can probably go in a future PR but I don't want to edit too much after approval 👍

Reply:
Feel free to delete. If it works, I’m game.

Reply:
Gonna merge so I can move past this particular PR but hopefully I'll be back and can clean it up then.